recipe.cake

```diff
@@ -438,7 +435,7 @@ BuildParameters.SetParameters(context: Context,
                             getMsisToSign: getMsisToSign,
                             getILMergeConfigs: getILMergeConfigs,
                             preferDotNetGlobalToolUsage: !IsRunningOnWindows(),
-                            shouldBuildMsi: true,
+                            shouldBuildMsi: false,

```

Comment:
Ah. Should this be dependent on something else? Cause this is completely not going to let the MSI get built, right?

Reply:
So this sets the default to be false so that someone just running the build locally doesn't need to worry about generating the msi. But in CI on TeamCity and GitHub Actions we are explicitly setting it to true, so that they will always generate an MSI.

Reply:
Plus, if we _really_ want to build the MSI locally, we can pass in the same command line option.

This line could have been removed completely, as `false` is the default value, but there is no harm in being explicit about it.